pkg/controller/nodenetworkstateconfiguration/nodenetworkstateconfiguration_controller.go

```diff
@@ -101,6 +105,63 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	return nil
 }
 
+func setCondtionFailed(instance *nmstatev1alpha1.NodeNetworkState, message string) {
+	conditions.SetCondition(
+		instance,
+		nmstatev1alpha1.NodeNetworkStateConditionFailing,
+		corev1.ConditionTrue,
+		"Failed",
+		message,
+	)
+	conditions.SetCondition(
+		instance,
+		nmstatev1alpha1.NodeNetworkStateConditionAvailable,
+		corev1.ConditionFalse,
+		"Failed",
+		message,
+	)
+}
+
+func setCondtionSuccess(instance *nmstatev1alpha1.NodeNetworkState, message string) {
+	conditions.SetCondition(
+		instance,
+		nmstatev1alpha1.NodeNetworkStateConditionAvailable,
+		corev1.ConditionTrue,
+		"Success",
+		message,
+	)
+	conditions.SetCondition(
+		instance,
+		nmstatev1alpha1.NodeNetworkStateConditionFailing,
+		corev1.ConditionFalse,
+		"Success",
+		message,
```

Comment:
maybe here we should just leave the message empty?

Reply:
I think it is worth mentioning why the Success is False (because of the error). However for the next comment I agree. Explaining why it is not Failing is not interesting.